slycot/analysis.py

```diff
@@ -221,69 +220,73 @@ def ab05md(n1,m1,p1,n2,p2,A1,B1,C1,D1,A2,B2,C2,D2,uplo='U'):
+        The number of input variables for the first system. m1 > 0.
+    p1 : int
+        The number of output variables from the first system and the number
+        of input variables for the second system. p1 > 0.
+    n2 : int
+        The number of state variables in the second system, i.e. the order
+        of the matrix A2.  n2 > 0.
+    p2 : int
+        The number of output variables from the second system. p2 > 0.
+    A1 : (n1, n1) array_like
+        The leading n1-by-n1 part of this array must contain the state
+        transition matrix A1 for the first system.
+    B1 : (n1, m1) array_like
+        The leading n1-by-m1 part of this array must contain the input/state
+        matrix B1 for the first system.
+    C1 : (p1, n1) array_like
+        The leading p1-by-n1 part of this array must contain the state/output
+        matrix C1 for the first system.
+    D1 : (p1, m1) array_like
+        The leading p1-by-m1 part of this array must contain the input/output
+        matrix D1 for the first system.
+    A2 : (n2, n2) array_like
+        The leading n2-by-n2 part of this array must contain the state
+        transition matrix A2 for the second system.
+    B2 : (n2, p1) array_like
+        The leading n2-by-p1 part of this array must contain the input/state
+        matrix B2 for the second system.
+    C2 : (p2, n2) array_like
+        The leading p2-by-n2 part of this array must contain the state/output
+        matrix C2 for the second system.
+    D2 : (p2, p1) array_like
+        The leading p2-by-p1 part of this array must contain the input/output
+        matrix D2 for the second system.
+    uplo :  {'U', 'L'}, optional
+        Indicates whether the user wishes to obtain the matrix A in
+        the upper or lower block diagonal form, as follows:
+            = 'U':  Obtain A in the upper block diagonal form;
+            = 'L':  Obtain A in the lower block diagonal form.
+        Default is `U`.
+
+    Returns
+    -------
+    n : int
+        The number of state variables (n1 + n2) in the resulting system,
+        i.e. the order of the matrix A, the number of rows of B and
+        the number of columns of C.
+    A : (n1+n2, n1+n2) ndarray
+        The leading N-by-N part of this array contains the state transition
+        matrix A for the cascaded system.
+    B : (n1+n2, m1) ndarray
+        The leading n-by-m1 part of this array contains the input/state
+        matrix B for the cascaded system.
+    C : (p2, n1+n2) ndarray
+        The leading p2-by-n part of this array contains the state/output
+        matrix C for the cascaded system.
+    D : (p2, m1) ndarray
+        The leading p2-by-m1 part of this array contains the input/output
+        matrix D for the cascaded system.
+
+    Notes
+    -----
         The implemented methods rely on accuracy enhancing square-root or
         balancing-free square-root techniques.
         The algorithms require less than 30N^3  floating point operations.
```

Comment:
Unindent here?

Reply:
Yup, was so before https://github.com/python-control/Slycot/blob/master/slycot/analysis.py .

I will change it to be unindent.

Reply:
Done.